src/types/request.ts

```diff
@@ -9,6 +9,7 @@ export type Orientation = 'landscape' | 'portrait' | 'squarish';
 export type OrientationParam = {
   orientation?: Orientation;
 };
+export type Plus = 'mixed' | 'only' | 'none';
```

Comment:
Do we want a `mixed` variant? If we look at the `Orientation` filter the `mixed` variant is implied by the absence of the orientation value. Shouldn't we mirror this convention for the plus param?

Reply:
Good question. I think we should keep `mixed` here because it matches API's behaviour as in `mixed` or `undefined` equals `mixed`. Orientation doesn't have any value encapsulating all orientations so we have to use `undefined`. I think it's a difference in design

Reply:
Makes sense. :+1:

Reply:
Here's the relevant slack thread: https://crewlabs.slack.com/archives/C4LRBSTG9/p1683568502846389 — I think third party apps already use the `plus` query params (we can track this because it needs special permissions) so any changes would be a breaking change.

Reply:
Ha! So the plus parameter was already available when you added the filter.

Reply:
If you're talking about unsplash.com filter, then it was about during the same time.